### ISSUE TITLE
deprecated filter hook message 

### ISSUE THREAD

**Issue by TheHeat** [sendgrid/wordpress#91]
Hi, I'm getting an alert in the dashboard that I think originates from line 52 of lib/class-sendgrid-settings.php

`// Add SendGrid Help contextual menu in the settings page
add_filter( 'contextual_help', array( __CLASS__, 'show_contextual_help' ), 10, 3 );`

The message is: 

> Deprecated: contextual_help is deprecated since version 3.3.0! Use get_current_screen()->add_help_tab(), get_current_screen()->remove_help_tab() instead. in /var/www/html/wp-includes/functions.php on line 5088

If I comment out line 52 it goes away.

**Comment by lkraav** [comment]
Relatively easily fixed. Y'all can take a page out of https://github.com/mailpoet/wp-mail-logging/pull/116 playbook.

**Comment by mitchelldmiller** [comment]
> mailpoet/wp-mail-logging#116

Does that mean this plugin / respository is not supported and will never be updated?

**Comment by lkraav** [comment]
> Does that mean this plugin / respository is not supported and will never be updated?

My comment was not meant to reflect on the development process of this plugin in any way.

That being said, last commit is from 2 years ago. Clearly not a high priority. There are other SMTP plugins, that are actively maintained and integrate SendGrid. I've already thought about migrating away from here but it's not a high priority for us either.

**Comment by RobertoDonPedro** [comment]
@lkraav Well, the mailpoet code does not work 1:1 for the sendgrid plugin. If you have solved this issue for the sendgrid plugin, would it please be possible for to share it? Thx.

**Comment by lkraav** [comment]
> Well, the mailpoet code does not work 1:1 for the sendgrid plugin. If you have solved this issue for the sendgrid plugin, would it please be possible for to share it? Thx.

Depends on what "does not work" means exactly.

Nothing bad really happens if you just comment out this `add_filter()` call, like pointed in the opening issue. I doubt any experienced user has a need for this help screen.

**Comment by RobertoDonPedro** [comment]
@lkraav Yes, that's what I did yet, commenting out the the add_filter hook. And you are right, the help screen is not really needed, however, if someone would have fully fixed it yet, it wouldn't hurt either :).
